contrib/zmq/zmq_sub.py

```diff
@@ -58,18 +57,18 @@ async def handle(self) :
             sequence = str(struct.unpack('<I', seq)[-1])
         if topic == b"hashblock":
             print('- HASH BLOCK ('+sequence+') -')
-            print(binascii.hexlify(body))
+            print(body.hex().encode())
```

Comment:
nit: Though this is right from a refactoring perspective, I don't think the `encode()` part is needed here (and in the substitutions below). This leads to displaying an example hex-string '1234abcd' as
```
b'1234abcd'
```
instead of
```
1234abcd
```
The latter seems more natural and was also the behaviour of `print(binascii.hexlify(...))` on Python 2.7 (which was used when this file was first introduced, see commit e6a14b64d665eb1fafd03a6bbc8d14597ce1c83c), when there was no 'bytes' type yet.

```suggestion
            print(body.hex())
```

Reply:
Ahh, makes sense now, the issue was wrongly assuming it was Py3 code to begin with :)

Fixed to replicate the original Py2 behavior (i.e. printing the hex-strings).